src/views/dialogs/MnemonicExportDialog.tsx

```diff
@@ -155,8 +155,7 @@ Styled.CautionIconContainer = styled.div`
     content: '';
     width: 2.5rem;
     height: 2.5rem;
-    background-color: var(--color-error);
-    opacity: 0.1;
+    background-color: var(--color-gradient-error);
   }
```

Comment:
slightly diff (`color-gradient-error` uses 0.16 opacity) but this is more in line with what is used in the rest of the app